.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Homebrew Release
+on:
+  workflow_dispatch:
+    inputs:
+      formula:
+        description: 'Formula to release Homebrew'
+        default: twilio
+      version:
+        description: 'Release version - Same as CLI version'
+        required: true
+      sha:
+        description: 'SHA of tar file created by oclif'
+      pre-release:
+        description: 'If it is a prerelease send true as input'
+        default: 'false'
```

Comment:
[Q] Do we need to manually trigger a release on Homebrew package for each release? Or does a twilio-cli/twilio-cli-core workflow trigger this workflow as well?

Reply:
We are triggering this workflow from [twilio-cli](https://github.com/twilio/twilio-cli/pull/287/files#diff-622ac52a2eb0f33afae0090ea57482ab4dbe959fe0b3ba158483889a2aceb074R52)